docs/kustomization.yaml

```diff
@@ -184,3 +184,20 @@ vars:
       apiVersion: apps/v1
     fieldref:
       fieldpath: spec.template.spec.restartPolicy
+
+# ImageTags are used to modify the tags for images used in different resources without creating patches.
```

Comment:
ImageTags modify the tags for images without creating patches.
E.g. Given this fragment of a Deployment: